pkg/squashfs/squashfs.go

```diff
@@ -254,10 +262,23 @@ func squashFuse(squashFile, extractDir string) (*exec.Cmd, error) {
 	log.Debugf("Extracting %s -> %s with squashfuse [%s]", squashFile, extractDir, logf)
 	err = cmd.Start()
 	if err != nil {
-		return nilCmd, err
+		return err
+	}
+
+	err = cmd.Process.Release()
```

Comment:
It's *probably* ok to do this, but this does mean that the squashfuse process will not be automatically reaped when stacker exits.  At least not by golang.  Presumably stacker has created its own namespace and that will cause it to be reaped anyway.

Reply:
ah. ok. so we could allow the caller to do the Release() if they wanted.
and that probably is better.